web/src/lib/components/album-page/album-options.svelte

```diff
@@ -26,11 +41,20 @@
 
   $: selectedOption = order ? options[order] : options[AssetOrder.Desc];
 
-  const handleToggle = async (returnedOption: RenderedOption) => {
+  // Fetch the current user when the component mounts
+  onMount(async (): Promise<void> => {
```

Comment:
There's no need for that, on the first page load, we save the info about the user in `$user`.  See in `+page.svelte` the `user={$user}`. You probably just want to rename `export let user: UserResponseDto;` to `export let currentUser: UserResponseDto;`